### ISSUE TITLE
Add data attributes that indicate the number of columns in a row

### ISSUE THREAD

**Issue by MoritzLost** [verbb/formie#1719]
### What are you trying to do?

When modifying the styling and responsive behaviour of Formie layouts, some layout methods require knowledge of the number of columns in a row. Related issue: #1718

It would be useful if Formie would output some `data-*` attributes or classes that indicate the number of fields (columns) in a form row. For example:

```html
<div class="fui-row fui-row-page" data-fui-field-count="3"></div>
```

This can be achieved to a degree using theme config, but only with a class, not a data attribute. I also haven't found a way to do the same for nested subfields.

```php
// config/formie.php
return [
    'themeConfig' => [
        'row' => [
            'attributes' => [
                'class' => [
                    'row-fields-{{ row.fields|length }}',
                ],
            ],
        ],
    ],
];
```

### What's your proposed solution?

Formie could output data attributes indicating the number of columns, both in regular rows as well as nested subfields.

**Comment by engram-design** [comment]
Added for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.1.3"`.

**Comment by MoritzLost** [comment]
@engram-design That's looking great, thanks!!

**Comment by engram-design** [comment]
Fixed in [2.1.4](https://github.com/verbb/formie/releases/tag/2.1.4)
